### ISSUE TITLE
Build solution

### ISSUE THREAD

**Issue by AleBar7** [opc-cpvp/OPC.PowerApps.PCFControls#17]
Hi,
i'm following the instructions to build the solution to load the component on a CRM organization. 
When i launch the last step 
![image](https://user-images.githubusercontent.com/39051818/89810138-dec20780-db3c-11ea-8e15-999c64f090dd.png)

i receive an error "Impossible to load the file or assembly 'bolt.module.solution, Version=1.3.0
.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of his dependencies. Impossible to fine the specified file. [C:\Users\username\Downloads\OPC.PowerApps.PCFControls-master\OPC.PowerApps.PCFControls-master\src\Solutio
n\Solution.cdsproj]"

**Comment by joshcampbell191** [comment]
Hello @AleBar7,

I was able to reproduce the issue when building from a fresh copy of the sources on my machine. I'll review the build process and see if there's any missing steps. In the meantime, I uploaded a copy of the managed / unmanaged solution to the project's [releases](https://github.com/opc-cpvp/OPC.PowerApps.PCFControls/releases/latest).

Once we've narrowed down the issue, we'll update the issue / instructions accordingly.

Cheers,
Joshua Campbell
